Description: add default pdf laucher to list of pdf viewers
 Upstream has a hardcoded list of programs to look for to show pdf documentation.
 However, on Debian systems we have xdg-open to open the user's default
 pdf viewer.
Author: Paul Gevers <elbrus@debian.org>
Forwarded: no

--- winff-1.5.1.orig/unit1.pas
+++ winff-1.5.1/unit1.pas
@@ -1340,10 +1340,12 @@ s:string;
 begin
   {$ifdef linux}
   s:='';
+  if fileexists('/usr/bin/okular') then s:='/usr/bin/okular';
   if fileexists('/usr/bin/evince') then s:='/usr/bin/evince';
   if fileexists('/usr/bin/kpdf') then s:='/usr/bin/kpdf';
   if fileexists('/usr/bin/xpdf') then s:='/usr/bin/xpdf';
   if fileexists('/usr/bin/acroread') then s:='/usr/bin/acroread';
+  if fileexists('/usr/bin/xdg-open') then s:='/usr/bin/xdg-open';
   if s='' then
      begin
      Showmessage('More information can be found at ' + pdffile);
